### PR TITLE
aligned images to center

### PR DESCRIPTION
**Pull request by shubhangi013**
solved #265 
Screenshot from 2022-03-30 13-05-13

### CODE REVIEW

**Review by TildaDares — CHANGES_REQUESTED** [review]

**Comment by shubhangi013** [comment]
@TildaDares I've done the requested changes :smile:

File: README.md [hunk 1]
```diff
@@ -12,9 +12,9 @@ Check it here: https://publiclab.github.io/infragram/
 This is ported out of the Node server application at https://github.com/p-v-o-s/infragram-js
 
 Read more at https://publiclab.org/infragram
-
-![split image showing a pre-converted blueish image on the left and rainbow-colored image on right](https://publiclab.org/system/images/photos/000/045/468/large/rgn-split.png)
-
+<div align="center">
+<img src='https://publiclab.org/system/images/photos/000/045/468/large/rgn-split.png'>
+</div>
```
[the remark this answers is not in the document]
Comment (TildaDares) [reply README.md:17]: @shubhangi013 The alt text is the text between the square brackets.

**Review by TildaDares — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (TildaDares) [reply README.md:17]: Look at line 16 of the code you removed.

**Comment by shubhangi013** [comment]
@TildaDares you can check now. :smile:

File: README.md [hunk 2]
```diff
@@ -153,9 +156,9 @@ Once these are selected, the pop-up disappears and the image is shown with these
 The processors also handle colorizing, which is also confusing because the `webgl` colorizer has to work very differently than the `javascript` version. 
 
 Colorizing is just converting a black and white (monochrome) image to a color one, where a given spectrum or "color map" is used to convert each pixel's brightness value to a color. The most familiar would be where "hot" colors like red represent higher values, and "cold" colors like blue represent lower values. The range of colors is shown in a key at the bottom of the image. 
-
-![screenshot of the color bar](https://publiclab.org/system/images/photos/000/045/470/medium/Screen_Shot_2022-03-08_at_2.56.38_PM.png)
-
+<div align="center">
+<img src='https://publiclab.org/system/images/photos/000/045/470/medium/Screen_Shot_2022-03-08_at_2.56.38_PM.png' alt='colorizing example'>
```
Comment (TildaDares) [inline README.md:160]:
```suggestion
<img src='https://publiclab.org/system/images/photos/000/045/470/medium/Screen_Shot_2022-03-08_at_2.56.38_PM.png' alt='screenshot of the color bar'>
```

**Review by TildaDares — CHANGES_REQUESTED** [review]

**Comment by shubhangi013** [comment]
@TildaDares I've done the suggested changes!

**Review by TildaDares — APPROVED** [review]
Great work @shubhangi013!!

**Pull request merged**